src/components/PeopleTable/SegmentAssignmentModal.jsx

```diff
@@ -0,0 +1,246 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import {
+  chakra,
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalHeader,
+  ModalFooter,
+  ModalBody,
+  Flex,
+  HStack,
+  VStack,
+  Box,
+  Button,
+  Text,
+  Tag,
+  TagLabel,
+  TagCloseButton,
+} from '@chakra-ui/react';
+import { ChevronDownIcon } from '@chakra-ui/icons';
+import { chakraComponents, Select } from 'chakra-react-select';
+
+const util = require('util');
+// import { OCHBackend } from '../../common/utils';
+
+// eslint-disable-next-line no-unused-vars
+const SegmentDropdown = ({ allSegments, selectedSegments, setSelectedSegments }) => {
+  const customSelectComponents = {
+    /* eslint-disable react/prop-types, react/jsx-props-no-spreading */
+    Menu: ({ children, ...props }) => (
+      <chakraComponents.Menu {...props}>
+        {children}
+        <chakraComponents.Option
+          {...props}
+          innerProps={{
+            sx: {
+              _hover: { bg: 'gray.100' },
+              _active: { bg: 'gray.200' },
+              d: 'flex',
+              w: '100%',
+              alignItems: 'center',
+              textAlign: 'center',
+              fontSize: 'md',
+              m: '0.3rem 0',
+              p: '0.4rem 0.8rem',
+              bg: 'white',
+              border: '1px solid',
+              borderRadius: '0.375rem',
+              borderColor: 'gray.100',
+              color: 'red.600',
+            },
+            onClick: e => {
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'e' is defined but never used.